content/Teams/observability-platform/sre.md

```diff
@@ -0,0 +1,119 @@
+
+*Deploying*
+* Use `synchronize.sh` to create an MR/PR against app-interface. This will copy all generated YAML resources to proper places.
+* Ask team for review. If change is impacting production heavily, notify AppSRE.
+* When merged, CI will deploy the change to production. You can see the version change on Monitoring dashboard too e.g https://prometheus.telemeter-prod-01.devshift.net/graph?g0.range_input=1h&g0.expr=thanos_build_info&g0.tab=0.
+
+## Incident Playbook Handling
+
+This is the process we as the Observability Team try to follow during incident response.
+
+NOTE: Following procedure applies to both Production and Staging. Many teams e.g SubWatch depends on working staging, so follow similar process as production. The only difference is that *we do not need to mitigate of fix staging issues outside of office hours*.
+
+*Potential Trigger Points*:
+
+* You got notified on Slack by AppSRE team or paged through Pager Duty.
+* You got notified about potential SLA violation by the customer: Unexpected responses, things that worked before do not work now etc.
+* You touch production for unrelated reasons and noticed something worrying (error logs, un-monitored resource etc).
+
+1. If you are not on-call notify Observability Platform on-call engineer. If you are on-call, on-call engineer is not present or you agreed that you will handle this incident, go to step 2.
+2. Straight away, create JIRA for potential incident. Don't think twice, it's easy to create and essential to track incident later on. Fill the following parts:
+* Title: Symptom you see.
+* Type: Bug
+* Priority: Try to assess how important it is. If impacting production it's a "Blocker"
+* Component: RHOBS
+* (Important) Label: `incident` `no-qe`
+* Description: Mention how you were notified (ideally with link to alert/Slack thread). Mention what you know so far.
+
+![jira](jira.png) See example incident tickets [here](https://issues.redhat.com/issues/?jql=project%20%3D%20MON%20AND%20labels%20%3D%20incident)
+
+1. If AppSRE is not yet aware, drop link to created incident ticket to #sd-app-sre channel and notify `@app-sre-primary` and `@observatorium-oncall`. Don't repeat yourself, ask everyone to follow the ticket comments.
+   1. AppSRE may or may not create dedicated channel, do communication efforts and start on-call Zoom meeting. We as the dev team don't need to worry about those elements, go to step 4.
+2. Investigate possible mitigation. Ensure the problem is mitigated before focusing on root cause analysis.
+   1. Important: Note all performed actions and observations on created JIRA ticket via comments. This allows anyone to follow up what was checked and how. It is also essential for detailed Post Mortem / RCA process later on.
+   2. Note on JIRA ticket all automation or monitoring gaps you wished you had. This will be useful for actions after the incident.
+3. Ensure if incident is mitigated with AppSRE. If yes, claim the incident being over.
```

Comment:
If we were being strict, I would say the incident is not 'over' until the root cause has been identified. WDYT?

Reply:
I would argue - if you mitigated = it's over.

For example: you might have no idea what's wrong with X, but you  rolled back ASAP and version X-1 works fine. You don't know the root cause of incident yet, but you know version X is causing some problem - yet incident is over technically.

WDYT?

Reply:
Version x being bad might be a root cause (: